src/views/v2/tickets/item.php

```diff
@@ -50,19 +55,17 @@
 	get_post_class( '', $ticket->ID ),
 ];
 
-$data_available      = 0 === $tickets_handler->get_ticket_max_purchase( $ticket->ID ) ? 'false' : 'true';
-$data_has_shared_cap = $has_shared_cap ? 'true' : 'false';
-$ticket_item_id      = 'tribe-';
-$ticket_item_id     .= ! empty( $is_modal ) ? 'modal' : 'block';
-$ticket_item_id     .= '-tickets-item-' . $ticket->ID;
+$ticket_item_id = 'tribe-';
+$ticket_item_id .= ! empty( $is_modal ) ? 'modal' : 'block';
+$ticket_item_id .= '-tickets-item-' . $ticket->ID;
 ?>
 <div
 	id="<?php echo esc_attr( $ticket_item_id ); ?>"
 	<?php tribe_classes( $classes ); ?>
 	data-ticket-id="<?php echo esc_attr( $ticket->ID ); ?>"
-	data-available="<?php echo esc_attr( $data_available ); ?>"
-	data-has-shared-cap="<?php echo esc_attr( $data_has_shared_cap ); ?>"
-	<?php if ( $has_shared_cap ) : ?>
+	data-available="<?php echo esc_attr( $this->get( 'data_available' ) ); ?>"
+	data-has-shared-cap="<?php echo esc_attr( $this->get( 'data_has_shared_cap' ) ); ?>"
+	<?php if ( $this->get( 'has_shared_cap' ) ) : ?>
```

Comment:
`$this->get()` isn't necessary when passed into the context, the vars are already passed/set.

Reply:
fixed